src/main/java/com/zontik/groshiky/controllers/AccountController.java

```diff
@@ -29,7 +29,7 @@ public AccountController(IAccountService accountService, IUserService userServic
     @PostMapping(value = "/addAccount")
     public ResponseEntity addAccount(Account account) {
```

Comment:
Ok for now, but any method in RestController will convert method return value to json. So Response entity will be converted too. I think it'll be better to throw an exception here and implement ExceptionHandling mechanism (but this manupulation better implement if you're planning to return a lot of response entities with status not equal than 200).